src/main/scala/it/pagopa/interop/backendforfrontend/server/impl/dependencies/Dependencies.scala

```
@@ -100,7 +100,24 @@ trait Dependencies {
     )
     .toFuture
 
-  def makeController(jwtReader: JWTReader, blockingEc: ExecutionContextExecutor)(implicit
+  def fileManager(blockingEc: ExecutionContextExecutor): FileManager =
+    FileManager.get(ApplicationConfiguration.storageKind match {
+      case "S3"   => FileManager.S3
+      case "file" => FileManager.File
+      case _      => throw new Exception("Incorrect File Manager")
+    })(blockingEc)
+
+  def getAllowList(blockingEc: ExecutionContextExecutor): Future[List[String]] = {
+    val filePath: String = s"${ApplicationConfiguration.allowListPath}/${ApplicationConfiguration.allowListFilename}"
+    fileManager(blockingEc)
+      .get(ApplicationConfiguration.allowListContainer)(filePath)
+      .map(byteStream => new String(byteStream.toByteArray).split('\n').flatMap(_.split(',')).toList)(blockingEc)
+      .recoverWith { case _ =>
+        Future.successful(Nil)
```

as discussed [here](https://github.com/pagopa/interop-be-backend-for-frontend/pull/76#discussion_r1069170088) in this case it's better to let it fail in case of errors